scriptmodules/helpers.sh

```diff
@@ -148,10 +153,10 @@ function inputBox() {
 
     if [[ -f "$osk" ]]; then
         params+=(--minchars "$minchars")
-        text=$(python3 "$osk" "${params[@]}" 2>&1 >/dev/tty) || return $?
+        text=$(python3 "$osk" "${params[@]}" "$text" 2>&1 >/dev/tty) || return $?
     else
         while true; do
-            text=$(dialog "${params[@]}" 10 60 "$text" 2>&1 >/dev/tty) || return $?
+            text=$(dialog "${params[@]}" 10 60 "Enter the $text" 2>&1 >/dev/tty) || return $?
```

Comment:
This is the default text so it should just be the `$text` variable as before. Eg for netplay.

![Screenshot_2023-02-25_17-54-49](https://user-images.githubusercontent.com/5365574/221372243-9f7f8767-42c9-4e6e-b6e7-940ef6ce08c4.png)

Reply:
I knew I forgot something...

Reply:
Should be fixed now, but I'll need to modify the OSK to remove the built-in _Enter the_. This is what I forgot.

Reply:
Ok, fixed the `osk.py` also, should be ok now.